atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -38,24 +38,21 @@ def __init__(self, workflow, ignore_autogenerated_gpg_keys=True):
         self.ignore_autogenerated_gpg_keys = ignore_autogenerated_gpg_keys
 
     def run(self) -> None:
+        if self.workflow.data.dockerfile_images.base_from_scratch:
+            self.log.info('Skipping plugin, no rpm(s) in FROM-scratch images')
+            return None
```

Comment:
I'm not sure if this is true. In fact, now that our base image builds look like this, it's most likely not true
```dockerfile
FROM scratch

ADD base-filesystem.tar.gz /
```

Previously, we would extract the RPMs first, and then, if there were none, check if the image is FROM scratch. I think we still need to do it that way

Reply:
If it's the right exception, yeah :smile: 

Might be good to give the extract method a parameter to allow the use case where nothing may get extracted

Reply:
1) extract rpmdb path 
2) check if rpmdb was extracted
3) if it was do query
4) if it wasn't , CHECK if build was FROM scratch , if was return NONE (as scratch are allowed not to have rpmdb)
but IF it has rpmdb (ubi-micro) do rpm query

Reply:
updated to follow this logic